src/Std/Time/Internal/UnitVal.lean

```diff
@@ -113,9 +110,9 @@ instance : LE (UnitVal α) where le x y := x.val ≤ y.val
 
 instance : LT (UnitVal α) where lt x y := x.val < y.val
 
-instance : Add (UnitVal α) where add x y := ⟨x.val + y.val⟩
+instance : Add (UnitVal α) where add := UnitVal.add
 
-instance : Sub (UnitVal α) where sub x y := ⟨x.val - y.val⟩
+instance : Sub (UnitVal α) where sub := UnitVal.sub
 
 instance : Mul (UnitVal α) where mul x y := ⟨x.val * y.val⟩
```

Comment:
This one should presumably use `.mul` too

Reply:
Actually, this and the instance below are both very suspicious; this allows nonsense like `60s*60s = 3600s`. I think this should be an HMul instance